pkg/exchange/coinbase/exchage.go

```diff
@@ -132,6 +136,16 @@ func (e *Exchange) SubmitOrder(ctx context.Context, order types.SubmitOrder) (cr
 		req.OrderType("limit")
 	case types.OrderTypeMarket:
 		req.OrderType("market")
+	case types.OrderTypeStopLimit:
+		req.OrderType("stop")
+		switch order.Side {
+		case types.SideTypeBuy:
+			req.Stop("entry") // trigger order when the price <= stop price
+		case types.SideTypeSell:
+			req.Stop("loss") // trigger order when the price >= stop price
```

Comment:
this is strategy dependent i think

Reply:
I see it over here:
https://docs.cdp.coinbase.com/exchange/reference/exchangerestapi_postorders#stop-orders

So that's why I think it's about stop-limit order under the context of bbgo.